tensorflow/lite/kernels/concatenation.cc

```diff
@@ -157,15 +157,17 @@ TfLiteStatus Eval(TfLiteContext* context, TfLiteNode* node) {
       TF_LITE_CONCATENATION(int32);
       break;
     case kTfLiteUInt8:
-      TF_LITE_CONCATENATION_QUANTIZED();
+      TF_LITE_CONCATENATION_QUANTIZED(uint8_t);
       break;
     case kTfLiteInt8:
       TF_LITE_CONCATENATION(int8_t);
```

Comment:
Can we instead make this code path shared with the int8 rather than the uint8 path. The uint8 path does rescaling whereas in the int8 we updated the code to require scales to match, and any rescaling of inputs should happen outside of the operation.